src/elements/primitives.rs

```diff
@@ -206,6 +206,43 @@ impl Serialize for VarInt7 {
     }
 }
 
+/// 8-bit unsigned integer, NOT encoded in LEB128;
+/// it's just a single byte.
+#[derive(Debug, Copy, Clone)]
+pub struct VarUint8(u8);
```

Comment:
Could you please rename it to `Uint8` to be more aligned with existing `Uint32`/`Uint64`?